### DISCUSSION TITLE
We should make a slack workspace

### DISCUSSION THREAD

**Discussion by ghost** [monkeytypegame/monkeytype#5176] [Ideas]
hear me out hear me out we should make a slack workspace for anyone willing to contribute it could help a lot of people understand the code better communicate etc. I know we already have a discord server but most programmers use slack and slack has integrated programming feutures

**Comment by Miodec** [comment]
We have a Discord server with a #development channel (discord.gg/monkeytype)

**Reply by ghost** [reply]
I know but it would be better with slack
